Example/EpoxyExample/ViewControllers/LayoutGroups/LayoutGroupsReadmeExamplesViewController.swift

```diff
@@ -0,0 +1,46 @@
+// Created by Tyler Hedrick on 2/5/21.
+// Copyright © 2021 Airbnb Inc. All rights reserved.
+
+import EpoxyCollectionView
+import UIKit
+
+class LayoutGroupsReadmeExamplesViewController: CollectionViewController {
+
+  // MARK: Lifecycle
+
+  init() {
+    super.init(layout: UICollectionViewCompositionalLayout.list)
+    setItems(items, animated: false)
+  }
+
+  // MARK: Internal
+
+  override var title: String? {
+    get { "Readme Examples" }
+    set { }
+  }
+
+  var items: [ItemModeling] {
```

Comment:
nit: can remove the nested `[…]` here too using `@ItemModelBuilder`
```suggestion
  @ItemModelBuilder
  var items: [ItemModeling] {
```